library/unwind/src/libunwind.rs

```diff
@@ -54,7 +54,7 @@ pub const unwinder_private_data_size: usize = 2;
 #[cfg(target_arch = "sparc64")]
 pub const unwinder_private_data_size: usize = 2;
 
-#[cfg(target_arch = "riscv64")]
+#[cfg(any(target_arch = "riscv64", target_arch = "riscv32")]
```

Comment:
```suggestion
#[cfg(any(target_arch = "riscv64", target_arch = "riscv32"))]
```

Reply:
Fixed. Sorry, I didn't do a build after the last change.